src/bp/index.ts

```diff
@@ -59,6 +59,9 @@ process.stderr.write = stripDeprecationWrite
 
 process.on('unhandledRejection', err => {
   global.printErrorDefault(err)
+  if (!process.IS_FAILSAFE) {
```

Comment:
I feel like this is safe, because we do the same check few lines down when handling `"uncaughtException"` and according to [this](https://nodejs.org/api/process.html#:~:text=In%20synchronous%20code%2C%20the%20'uncaughtException,list%20of%20unhandled%20rejections%20shrinks.), then only difference between both handles are that `"unhandledRejection"` handles errors that were thrown asynchronously (or rejected).

Why would both case be handled differently?